pkg/scalers/kafka_scaler.go

```diff
@@ -26,10 +26,11 @@ type kafkaScaler struct {
 }
 
 type kafkaMetadata struct {
-	bootstrapServers []string
-	group            string
-	topic            string
-	lagThreshold     int64
+	bootstrapServers    []string
+	group               string
+	topic               string
+	lagThreshold        int64
+	consumerOffsetReset offsetResetPolicy
```

Comment:
can't we find a better name instead of `consumerOffsetReset`. Actually the type name would be a great name for the struct field as well so `offsetResetPolicy`. It's not the first time that a field name is the same as the type name, or?

Reply:
Absolutely ;)